private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -83,12 +87,12 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		&f.Type,
 		typeFlagName,
 		"",
-		`The full type name of the serialized message (like acme.weather.v1.Units).
+		`The full type name of the serialized message (like acme.weather.v1.Units) within the input.
 Alternatively, this can be a fully qualified path to the type without providing the source (like buf.build/acme/weather#acme.weather.v1.Units).`,
 	)
 	flagSet.StringVar(
 		&f.Input,
```

Comment:
can we also rename this to payload?

Reply:
yeah

Reply:
I mean the `&f.Input` here, sorry if that is confused

Reply:
This actually is the "Input" of proto/image/bsr fqn so I think it works